packages/api/tests/endpoints/test-granules.js

```diff
@@ -1976,6 +1979,113 @@ test.serial('PUT replaces an existing granule in all data stores', async (t) =>
   );
 });
 
+test.serial('PUT creates a granule if one does not already exist in all data stores', async (t) => {
+  const {
+    knex,
+  } = t.context;
+
+  const granuleId = `${cryptoRandomString({ length: 7 })}.${cryptoRandomString({ length: 20 })}.hdf`;
+
+  const fakeGranule = fakeGranuleFactoryV2({
+    granuleId,
+    status: 'completed',
+    execution: t.context.executionUrl,
+    duration: 47.125,
```

Comment:
the collection created from the `fakeGranuleFactoryV2` record happens to have the same collection name and version as the collection specific in the test setup 
```
  const collectionName = 'fakeCollection';
  const collectionVersion = 'v1';
```

but if these were somehow to change in the fakeGranuleFactoryV2 function, than the call here 
```
  const actualPgGranule = await t.context.granulePgModel.get(knex, {
    granule_id: actualGranule.granuleId,
    collection_cumulus_id: t.context.collectionCumulusId,
  });
```
could be getting the wrong granule, right? Not a big deal but I think we could explicitly pass in the collection name and version here instead of relying on the values defined in `fakeGranuleFactoryV2`